Fully Kiosk Browser: migrate actions to dedicated action pages (#46170)
home-assistant/home-assistant.github.io: source/_actions/fully_kiosk.load_url.markdown

---
title: "Load URL"
action: fully_kiosk.load_url
domain: fully_kiosk
description: "Loads a URL on Fully Kiosk Browser."
related_actions:
  - fully_kiosk.set_config
  - fully_kiosk.start_application
---

> The **Load URL** action tells Fully Kiosk Browser to open a specific web address on your device.

Reviewer: Remove inconsistent UI action name, making it general:
```suggestion
This action tells Fully Kiosk Browser to open a specific web address on your device.
```

4. In the **Then do** section, select **Add action**.
5. From the search box, search for and select **Fully Kiosk Browser: Load URL**.
6. Select the **Device ID** to load the URL on.
7. Enter the **URL** to load.
8. Select **Save**.

### Options in the UI

{% options_ui %}
Device ID:
  description: The device running Fully Kiosk Browser to load the URL on.
  required: true
URL:
  description: The URL to load.
  required: true
{% endoptions_ui %}

{% include actions/yaml_header.md %}

In YAML, refer to this action as `fully_kiosk.load_url`:

{% example %}
action: |
  action: fully_kiosk.load_url
  data:
    device_id: a674c90eca95eca91f6020415de07713
    url: "https://www.home-assistant.io"
{% endexample %}

This opens the Home Assistant website on the selected device.

### Options in YAML

{% options_yaml %}
device_id:
  description: The device running Fully Kiosk Browser to load the URL on.
  required: true
  type: string
url:
  description: The URL to load.
  required: true
  type: string
{% endoptions_yaml %}

{% include actions/try_it.md %}

{% include actions/more_examples.md %}

> 

Reviewer: Remove empty section, that can be added later:
```suggestion
```
